src/ft2font.cpp

```diff
@@ -41,6 +42,12 @@
 
 FT_Library _ft2Library;
 
+void throw_ft_error(std::string message, int error) {
```

Comment:
Is there a reason not to use `FT_Error` instead of `int`?

Reply:
nope, changed it to FT_Error.